website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -166,6 +166,10 @@ These are the buckets that will be created in cumulus OU account:
 - `<PREFIX>-protected`
 - `<PREFIX>-tf-state` (for storing the terraform state file in cumulus OU account)
 
+:::tip
+The `*-tf-state` buckets and dynamoDB tables will not be automatically removed by cleanup scripts.
+Once you are done with your testing, and you have verified that cleanup is actually successful, manually delete these resources.
+:::
 
 After hitting the play button on `Deploy ORCA Buckets and Cumulus/ORCA modules`, but before hitting `Run` in the popup, replace the following variables with yours.

```

Comment:
based on your lastest changes, 
change to 
`Deploy Cumulus buckets and Cumulus and Orca modules`

Reply:
Updated